### ISSUE TITLE
Cylinder with Dip and Direction - Feature Request

### ISSUE THREAD

**Issue by Kapanther** [minorua/Qgis2threejs#218]
Be great to be able to specify a dip and dip direction to the cylinders, similar to the way the disk works...  then i can rotate cylinders for boireholes etc for inclined holes!!

![image](https://user-images.githubusercontent.com/12637127/81640450-ea306480-9461-11ea-9d1e-d394e36468e4.png)

**Comment by minorua** [comment]
Yeah, with this options, you don't need to create 3D lines along the inclined holes.

Maybe we also need vertical shift amount of cylinder to represent multiple segments.
